test_runner/regress/test_recovery.py

```diff
@@ -17,6 +17,8 @@ def test_pageserver_recovery(neon_env_builder: NeonEnvBuilder):
 
     neon_env_builder.start()
 
+    env.pageserver.allowed_errors.append(".*found future delta layer.*")
```

Comment:
```suggestion
    # These warnings are expected, when the pageserver is restarted abruptly
    env.pageserver.allowed_errors.append(".*found future image layer.*")
    env.pageserver.allowed_errors.append(".*found future delta layer.*")
```

If this was intentional, please add a comment why we only expect `delta` but not `image` layer error here.

Reply:
It was not intentional, applied your suggestion